frontstage/views/passwords/reset_password.py

```diff
@@ -17,19 +17,18 @@
 def get_reset_password(token, form_errors=None):
     form = ResetPasswordForm(request.form)
 
-    url = app.config['VERIFY_PASSWORD_TOKEN']
-    parameters = {"token": token}
-    response = api_call('GET', url, parameters=parameters)
-
-    if response.status_code == 409:
-        logger.warning('Token expired', token=token)
-        return render_template('passwords/password-expired.html')
-    elif response.status_code == 404:
-        logger.warning('Invalid token sent to party service', token=token)
-        return redirect(url_for('error_bp.not_found_error_page'))
-    elif response.status_code != 200:
-        logger.error('Party service failed to verify token')
-        raise ApiError(response)
+    try:
+        party_controller.verify_token(token)
+    except ApiError as exc:
+        if exc.status_code == 409:
+            logger.warning('Token expired', token=token)
```

Comment:
Is there any risk in logging out this token?

Reply:
@andrewmil I'm not sure. 

Team, @ONSdigital/sdc-ras, what do we think?

Reply:
Should be alright, It's expired. And if it's not okay, we have a bigger issue with sending the token in the url.

Reply:
Rob seems to think these tokens are fine to be logged out